src/main/java/entity/Team.java

```diff
@@ -0,0 +1,54 @@
+package entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@Entity
+@Data
+@AllArgsConstructor
+
+public class Team {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private static long idCounter = 1;
+    private long teamID;
```

Comment:
```suggestion
    private long id;
```
nombralo como id, recuerda DRY